### PR TITLE
Fix INVALID_REF errors caused by mods

### PR DESCRIPTION
**Pull request by ASpoonPlaysGames**
INVALID_REF errors were being caused by trying to get the item image for weapon skins. 

If a mod (moreskins) added new skins, the client equipped one, and then the mod is disabled, the client now has bad persistence.
This is mostly handled fine by the game, except for weapon skins. This PR prevents the crash, and resets the bad persistence to default.

## Testing:

- Install [MoreSkins](https://northstar.thunderstore.io/package/Creamy_jpg/MoreSkinsMinus/) and equip a weapon warpaint that does not exist in vanilla (example being masterwork for the r201)
- Make sure your persistence has saved (close the game and reopen it, validate that the skin is still there)
- Delete/disable MoreSkins
- Go to the loadout, the weapon should now have Factory Issue equipped

### Note: This PR introduces two vanilla files, `menu_pilot_loadouts_shared.nut` and `menu_edit_pilot_loadouts.nut` to `Northstar.Client/mod/scripts/vscripts/ui`, they will need to be committed as Respawn.

### CODE REVIEW

**Comment by F1F7Y** [comment]
Followed instructions and the skin defaults to factory issue even when MoreSkins is enabled:
image

After disabling the mod:
image

**Comment by ASpoonPlaysGames** [comment]
tf... ill look into that tomorrow then

**Comment by ASpoonPlaysGames** [comment]
> Followed instructions and the skin defaults to factory issue even when MoreSkins is enabled:

That looks normal? I probably should've mentioned that the actual gun skin looks like default, it's the icon thing you should be looking at

**Comment by ASpoonPlaysGames** [comment]
> After disabling the mod:
> image

I can't replicate this? Did you have any other mods installed when testing this that might've been causing issues?

**Comment by F1F7Y** [comment]
> I can't replicate this? Did you have any other mods installed when testing this that might've been causing issues?

Flightcore overwrote the folder so i cant check, Ill test it again tho

**Comment by ASpoonPlaysGames** [comment]
> Flightcore overwrote the folder so i cant check, Ill test it again tho

I would normally just add a check to prevent this crash, but this code needs to run, or you'll just get a crash later on. I might throw an error that's a bit more descriptive though

**Comment by F1F7Y** [comment]
> I can't replicate this? Did you have any other mods installed when testing this that might've been causing issues?

Having the masterwork skin selected and going into the "Pilot loadouts" menu with MoreSkins disabled causes this

**Comment by ASpoonPlaysGames** [comment]
> Having the masterwork skin selected and going into the "Pilot loadouts" menu with MoreSkins disabled causes this

Still can't replicate on my end unfortunately

**Review by F1F7Y — APPROVED** [review]
Couldn't manage to break it, changes look good

NOTE: Adds vanilla files

File: Northstar.Client/mod/scripts/vscripts/ui/menu_pilot_loadouts_shared.nut
Comment (GeckoEidechse) [inline Northstar.Client/mod/scripts/vscripts/ui/menu_pilot_loadouts_shared.nut:1]: Vanilla file, needs to be committed as Respawn first by a maintainer.

**Review by GeckoEidechse — COMMENTED** [review]
@ASpoonPlaysGames from which VPK are the files from? ^^

**Comment by ASpoonPlaysGames** [comment]
> @ASpoonPlaysGames from which VPK are the files from? ^^

englishclient_frontend

**Comment by F1F7Y** [comment]
@GeckoEidechse bump

**Review by uniboi — APPROVED** [review]
please please please the code is good and does what it's supposed to

File: Northstar.Client/mod/scripts/vscripts/ui/menu_edit_pilot_loadouts.nut
[the remark this answers is not in the document]
Comment (uniboi) [reply Northstar.Client/mod/scripts/vscripts/ui/menu_edit_pilot_loadouts.nut:1]: then just do that where is the problem

[the remark this answers is not in the document]
Comment (GeckoEidechse) [reply Northstar.Client/mod/scripts/vscripts/ui/menu_edit_pilot_loadouts.nut:1]: Done in 7893a11faa5a9f3d13b241b1269b169ae8c519d3

File: Northstar.Client/mod/scripts/vscripts/ui/menu_pilot_loadouts_shared.nut
Comment (GeckoEidechse) [reply Northstar.Client/mod/scripts/vscripts/ui/menu_pilot_loadouts_shared.nut:1]: Done in c2716db187ce1e896031ac75e097f96754dc92cf

**Comment by GeckoEidechse** [comment]
Merging based on past reviews.

**Pull request merged**